src/common/options.cc

```diff
@@ -431,6 +431,11 @@ std::vector<Option> get_global_options() {
     .add_tag("network")
     .add_see_also("mon_host"),
 
+    Option("image", Option::TYPE_STR, Option::LEVEL_BASIC)
```

Comment:
Ouch -- this totally breaks the `rbd` CLI since it now strips the `--image` optional out since it's now a Ceph global config override. Can we rename this to something less generic like `container_image`?